test/compile_test.sh

```diff
@@ -290,3 +290,13 @@ testComplile_BuildPropertiesFileWithMServerVersion()
 
   assertContains "SBT should have been installed" "Downloading sbt launcher for" "$(cat ${STD_ERR})"
 }
+
+testComplile_CreatesExportScript()
+{
+  createSbtProject
+
+  compile
+
+  assertEquals 0 "${RETURN}"
+  assertTrue "Export script should be created" "[ -f export ]"
```

Comment:
You could additionally execute this script and check the exported vars if you wanted stronger tests

Ex: https://github.com/heroku/heroku-buildpack-nodejs/blob/master/test/run#L956-L1004